### PR TITLE
Update the file metadata for all the GitOps assemblies and modules to…

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**Aligned team**: Dev Tools

**Please cherry-pick to**: `gitops-docs-1.11`. `gitops-docs-1.10`, and `gitops-docs-1.9`

**JIRA issues**: [RHDEVDOCS-5797](https://issues.redhat.com/browse/RHDEVDOCS-5797)

**Preview pages**: 
- [Adding permissions for cluster configuration](https://70044--docspreview.netlify.app/openshift-gitops/latest/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations#gitops-additional-permissions-for-cluster-config_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)
- [Installing an Argo CD instance](https://70044--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance#gitops-argo-cd-installation_setting-up-argocd-instance)
- [Argo CD custom resource properties](https://70044--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties#argo-cd-properties_argo-cd-cr-component-properties)
- [Repo server properties](https://70044--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties#argo-repo-server-properties_argo-cd-cr-component-properties)
- [Default permissions of an Argocd instance](https://70044--docspreview.netlify.app/openshift-gitops/latest/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations#default-permissions-of-an-argocd-instance.adocconfiguring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)
- [In-built permissions for cluster configuration](https://70044--docspreview.netlify.app/openshift-gitops/latest/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations#gitops-inbuilt-permissions-for-cluster-config_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)
- [Using an Argo CD instance to manage cluster-scoped resources](https://70044--docspreview.netlify.app/openshift-gitops/latest/declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations#using-argo-cd-instance-to-manage-cluster-scoped-resourcesconfiguring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations)

**SME Review**: Not applicable

**QE review**: Not applicable

**Peer review**:  

**Additional information**:  This PR updates the file metadata for all the GitOps assemblies and modules to reflect the latest `:_mod-docs-content-type` as per the OCP docs guidelines. In addition, this PR replaces double asterisks with single asterisks for web console / GUI items (menus, buttons, page titles, etc.) as per the OCP docs guidelines.

### CODE REVIEW

File: modules/gitops-default-permissions-of-an-argocd-instance.adoc [hunk 1]
```diff
@@ -2,13 +2,13 @@
 //
 // * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
 
-:_content-type: PROCEDURE
+:_mod-docs-content-type: PROCEDURE
 [id="default-permissions-of-an-argocd-instance.adoc{context}"]
 = Default permissions of an Argocd instance
```
Comment (tmalove) [inline modules/gitops-default-permissions-of-an-argocd-instance.adoc:7]:
```suggestion
= Default permissions of an Argo CD instance
```

Comment (Srivaralakshmi) [reply modules/gitops-default-permissions-of-an-argocd-instance.adoc]: @tmalove Thanks for the feedback. Good catch! Addressed it.

**Comment by Srivaralakshmi** [comment]
Thank you, @tmalove for your approval.

**Pull request merged**